### ISSUE TITLE
Add references to the original docs

### ISSUE THREAD

**Issue by vcwild** [vcwild/qa4noobs#3]
# Add references to the original docs

The references should be added to a single REFERENCES.md file.

e.g.

```md
# References

Text

## First topic

Text

### First subtopic

Text
- Bullet point
- Bullet point 2

```

**Comment by vcwild** [comment]
@VictorMPicoli we agreed that the references should be done by chapter, I will create an issue just for the first one

**Comment by vcwild** [comment]
Depends on #4

**Comment by vcwild** [comment]
Closed as won't do
